scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -168,10 +168,10 @@ task CreateImportTsvs {
         --mode GENOMES \
         -SNM ~{sample_map} \
         --ref-version 38
-        
-      gsutil cp metadata_*.tsv ~{output_directory}/metadata_tsvs/
-      gsutil cp pet_*.tsv ~{output_directory}/pet_tsvs/
-      gsutil cp vet_*.tsv ~{output_directory}/vet_tsvs/
+
+      gsutil -m cp metadata_*.tsv ~{output_directory}/metadata_tsvs/
```

Comment:
Did you ever determine the change in behavior when adding the -m w.r.t exit codes?

For example, trying `gsutil cp file1.txt filethatdoesntexist.txt gs://destination` vs the same thing with -m.  We can likely handle either case, but we should know (and maybe put a comment here about it)

Reply:
yes, I addressed this in the PR comment! but I can add a comment to the code, sure